### DISCUSSION TITLE
Personnaliser vos animations avec flutter_animate

### DISCUSSION THREAD

**Discussion by giscus** [zero-filtre/zerofiltre-blog#237] [Announcements]
# Personnaliser vos animations avec flutter_animate

Cet article explique comment personnaliser les animations dans les applications Flutter en utilisant la bibliothèque flutter_animate. Il couvre l'installation, l'utilisation de base, et diverses techniques de personnalisation.Soyez servi

https://zerofiltre.tech/articles/304-personnaliser-vos-animations-avec-flutter_animate

**Comment by imphilippesimo** [comment]
Il y'a des fautes d'orthographe et de grammaire.
Passez chacun de vos paragraphes à chatgpt ou Gemini afin qu'ils soient corrigés.

Nous avons [une procédure](https://github.com/Zerofiltre-Courses/articles/blob/main/README.md) qui décrit comment faire référencer le code source de son
article. Veuillez la respecter.
